server/room-minor-activity.ts

```diff
@@ -95,7 +95,10 @@ export abstract class MinorActivity {
 			});
 
 			if (!MinorActivityClass) {
-				if (pollData.activityid === 'poll') throw new Error('No minorActivity class provided');
```

Comment:
I think I want to know more about why `MinorActivityClass` doesn't exist here. It definitely looks like we need some sort of global registry of minor activity classes (probably also games) to make this easier to use.